virtcontainers/qemu_s390x.go

```diff
@@ -257,3 +257,30 @@ func (q *qemuS390x) appendSCSIController(devices []govmmQemu.Device, enableIOThr
 	devices = append(devices, d)
 	return devices, t
 }
+
+func (q *qemuS390x) appendVSock(devices []govmmQemu.Device, vsock kataVSOCK) []govmmQemu.Device {
+	var devno string
+	id := fmt.Sprintf("vsock-%d", vsock.contextID)
+	addr, b, err := q.addDeviceToBridge(id, types.CCW)
+        if err != nil {
+                logrus.Errorf("Failed to append VSock: %v", err)
```

Comment:
The two error scenarios here imply to me that `appendVSock()` should return an error (now):

```go
appendVSock(devices []govmmQemu.Device, vsock kataVSOCK) ([]govmmQemu.Device error)
```

Reply:
@jodh-intel good point! I updated the PR. Actually, we need to add an error for all the append function introduced by 5f2c193d . Is it ok if I open another PR to fix that?

Reply:
Hi @alicefr - sure :smile: